src/w3c/templates/show-people.js

```diff
@@ -12,23 +12,22 @@ export default (conf, name, items = []) => {
       data-editor-id='${editorid}'></dd>`;
       const span = document.createDocumentFragment();
     const contents = [];
-    if (p.url) {
+    if (p.mailto) {
+      contents.push(html`<a class='ed_mailto u-email email p-name'
+        href='${`mailto:${p.mailto}`}'>${p.name}</a>`);
+    } else if (p.url) {
       contents.push(html`<a class='u-url url p-name fn'
         href='${p.url}'>${p.name}</a>`);
     } else {
       contents.push(html`<span class='p-name fn'>${p.name}</span>`);
     }
     if (p.company) {
       if (p.companyURL) {
-        contents.push(html`, <a class='p-org org h-org h-card' href='${p.companyURL}'>${p.company}</a>`);
+        contents.push(html` (<a class='p-org org h-org h-card' href='${p.companyURL}'>${p.company}</a>)`);
       } else {
-        contents.push(document.createTextNode(`, ${p.company}`));
+        contents.push(document.createTextNode(` (${p.company})`));
```

Comment:
Would "html` (${p.company})`" not do the same thing (i.e. create the text node)?